cmd/controller/app/options/options.go

```diff
@@ -80,6 +80,7 @@ type ControllerOptions struct {
 	ACMEHTTP01SolverResourceRequestMemory string
 	ACMEHTTP01SolverResourceLimitsCPU     string
 	ACMEHTTP01SolverResourceLimitsMemory  string
+	RunAsNonRoot                          bool
```

Comment:
suggestion: as with the name of the command line flag, do you think we could be more descriptive here in this name (and in `defaultRunAsNonRoot`) about what would be the thing running as non-root?